### DISCUSSION TITLE
Will there be any overlap of zeek logs?

### DISCUSSION THREAD

**Discussion by xStevex** [Security-Onion-Solutions/securityonion#5458] [Unsupported Versions]
Hi there.
We have Zeek sensors on all egress points that send logs to the Zeek server (Zeek was deployed as a cluster).
And if I will install Security Onion that has build-in Zeek. Will there be any overlap of Zeek logs?
Is it possible to collect Zeek logs from Zeek server /opt/zeek/logs/*?
And will there be no conflict with the built-in Zeek sensor?

**Comment by dougburks** [comment]
> We have Zeek sensors on all egress points that send logs to the Zeek server (Zeek was deployed as a cluster).
> And if I will install Security Onion that has build-in Zeek. Will there be any overlap of Zeek logs?

If Security Onion and your separate Zeek boxes are monitoring the same traffic, then they will generate the same Zeek logs.

> Is it possible to collect Zeek logs from Zeek server /opt/zeek/logs/*?
> And will there be no conflict with the built-in Zeek sensor?

It's possible but we don't provide any support for it. Also keep in mind that you would then be managing two separate deployments. Whereas if you simply use Zeek built into Security Onion, then you just have one deployment to manage and it can all be done in one central location.
